### DISCUSSION TITLE
examples with just Arduino Uno and RPi

### DISCUSSION THREAD

**Discussion by HaydarJawad** [eclipse-ditto/ditto#1527] [General]
Hi guys, any chance you can do examples with just Arduino Uno and RPi with simple sensors directly attached without any specialist boards like Octopus board or GRovePi+.
Thank you

**Comment by thjaeckle** [comment]
We'll happily accept a PR providing such examples to the [ditto-examples](https://github.com/eclipse-ditto/ditto-examples) project.

**Comment by desislava-marinova** [comment]
@BabylonCoder , did you have a look at the [arduino-via-hono examples](https://github.com/eclipse-ditto/ditto-examples/tree/master/arduino-via-hono)? There are example with [mkr_wifi_1010](https://github.com/eclipse-ditto/ditto-examples/tree/master/arduino-via-hono/examples/mkr_wifi_1010) board and that in my opinion could be easily reused.

**Comment by HaydarJawad** [comment]
Hi , Yes i have i have but the explanation is in no way as clear as the RPI
Grove example. I can follow the Raspberry Pi and a GrovePi+ perfectly, but
because the Grove is no good. I was planning to teach it to my students
based on simple H/W that is available easily.  Thank you
